tracing-subscriber/src/filter/env/mod.rs

```diff
@@ -204,6 +204,22 @@ pub struct EnvFilter {
     regex: bool,
 }
 
+/// Creates an [`EnvFilter`] with the same directives as `self`. This does *not* clone any of the
```

Comment:
Sorry for another review round but can you puz the second sentence in its own paragraph? Just noticed this.